src/Core/Managed/Shared/Extensibility/Implementation/Metrics/MetricTerms.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+
+    internal static class MetricTerms
+    {
+        private const string MetricPropertiesNamePrefix = "Microsoft.ApplicationInsights.Metrics";
+
+        public static class Aggregation
+        {
+            public static class Interval
+            {
+                public static class Moniker
```

Comment:
I d not want to use namespaces here as discussed on the other thread. If you guys object to this stricture naming I can use just constants with long names instead if you like. :) However, it seems to me like a minor point.